source/_integrations/shelly.markdown

```diff
@@ -127,6 +127,14 @@ The integration uses the following strategy to name its entities:
 - If `Channel Name` is set in the device, the integration will use it to generate the entities' name, e.g. `Kitchen Light`
 - If `Channel Name` is set to the default value, the integration will use the `Device ID` and default channel name to generate the entities' name, e.g. `ShellyPro4PM-9808D1D8B912 switch_0`.
 
+## Cover entities
+
+Shelly 2PM Gen3 supports `tilt` for `cover` entities. To enable this feature, you need to configure the device:
+
+- change the device profile to `Cover` (**Settings** >> **Device profile**)
+- calibrate the cover (**Home** >> **Cover 0** >> **Calibration** >> **Start**)
+- enable and configure **Slat control** (**Home** >> **Cover 0** >> **Slat control**)
```

Comment:
Maybe we should remove the `0` since the dual cover will have two cover components